src/fabric/src/fabric2_server.erl

```diff
@@ -212,14 +213,47 @@ get_db_and_cluster(EunitDbOpts) ->
         {ok, true} ->
             {<<"eunit_test">>, erlfdb_util:get_test_db(EunitDbOpts)};
         undefined ->
-            ClusterFileStr = config:get("erlfdb", "cluster_file", ?CLUSTER_FILE),
+            ClusterFileStr = get_cluster_file_string(),
             {ok, ConnectionStr} = file:read_file(ClusterFileStr),
             DbHandle = erlfdb:open(iolist_to_binary(ClusterFileStr)),
             {string:trim(ConnectionStr), DbHandle}
     end,
     apply_tx_options(Db, config:get(?TX_OPTIONS_SECTION)),
     {Cluster, Db}.
 
+get_cluster_file_string() ->
+    Locations = [
+        {custom, config:get("erlfdb", "cluster_file", false)},
```

Comment:
As a minor nit, I wonder if we could switch to use `undefined` since in more cases in our code we rely on `config:get(Section, Key)` returning `undefined` but are not as familiar with `os:getenv/1`. In order to use `undefined` we could use [`os:getenv/2`](https://erlang.org/doc/man/os.html#getenv-2) as in `os:getenv("FDB_CLUSTER_FILE", undefined)` then handle the `undefined` in `find_cluster_file` clauses.

Reply:
I had started with that approach. Technically it violates the spec for `os:getenv/2` to use an atom as the second argument, but it does work. Will check if dialyzer complains.

Reply:
Ah makes sense then, if the dializer complains let's keep it as is. Maybe with a short comment about it?